package.json

```diff
@@ -17,6 +17,7 @@
     "@emotion/styled": "^11.10.5",
     "@portabletext/react": "^3.0.4",
     "@sanity/image-url": "^1.0.2",
+    "antd": "^5.8.2",
```

Comment:
Let's not use antd. We can't use two different UI Libraries in one project.

Also, you can use the useDisclosure hook with the chakra popover to trigger a hover effect to open it
They have example here https://chakra-ui.com/docs/components/popover/usage [controlled usage]

Reply:
can we stick with antd design for the hover in the meantime whiles we look for a way to use chakra